torchgeo/transforms/transforms.py

```diff
@@ -53,7 +53,7 @@ def __init__(
             else:
                 keys.append(key)
 
-        self.augs = K.AugmentationSequential(*args, data_keys=keys, **kwargs)
+        self.augs = K.AugmentationSequential(*args, data_keys=keys, **kwargs)  # type: ignore[arg-type] # noqa: E501
```

Comment:
Why is arg-type ignored here?

Reply:
We use kornia.contrib.Lambda for a few operations in the transforms functions, so the linter was getting upset about that. But there isn't another way to do those operations.

Reply:
It should be relatively easy to create a custom transform that subclasses the right class instead of using a Lambda. But let me report this typing bug to our Kornia contacts, should be really simple to fix.